src/pybkgmodel/data.py

```diff
@@ -654,25 +662,20 @@ def load_events(cls, file_name):
                             event_data[name] *= u.one
 
                 # Event times need to be converted from Instrument reference epoch
-                ref_epoch = astropy.time.Time(evt_head['MJDREFI']+evt_head['MJDREFF'], format='mjd')
+                ref_epoch = astropy.time.Time(evt_head['MJDREFI'], evt_head['MJDREFF'], format='mjd')
 
-                event_data['mjd'] = astropy.time.Time((evt_data['TIME'].to_numpy() 
-                                                       + ref_epoch.unix),
-                                                      scale='utc',
-                                                      format='unix'
+                event_data['mjd'] = astropy.time.Time((evt_data['TIME'].quantity 
```

Comment:
The outer time here is unnecessary.

The result of `ref_epoch + evt_data["TIME"].quantity` is a time, you are just making a copy again.

Reply:
Removed